pkgs/tools/games/slipstream/wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+_base="$HOME/.local/share/slipstream"
+
+mkdir -p "$_base"
+mkdir "$_base/mods" # FIX: slipstream needs mods dir to exist
+mkdir "$_base/backup"
+
+cd "$_base"
+
+java -jar "$jar_file"
```

Comment:
Is this really necessary ?

Reply:
It's the only way I was able to get it to work; Slipstream checks if a few directories exist in the directory it's run (namely the mods directory). They can't be in the nix store since they wouldn't be writable to add mods.

I'd be 100% okay with changing the paths or doing something else; I was just trying to get it to work and judging by people debugging Slipstream online this is what I deemed best. :)

Reply:
It seems Slipstream is indeed working like this. Maybe you could add a comment to explain / point a line in the source code? Looks like it is around here: https://github.com/Vhati/Slipstream-Mod-Manager/blob/master/src/main/java/net/vhati/modmanager/cli/SlipstreamCLI.java#L105